src/App/Pages/Accounts/LoginPasswordlessPage.xaml

```diff
@@ -19,62 +19,66 @@
                          x:Name="_closeItem" x:Key="closeItem" />
         </ResourceDictionary>
     </ContentPage.Resources>
-
-    <ScrollView Padding="7, 0, 7, 20">
-        <StackLayout>
-            <Label
-                Text="{u:I18n AreYouTryingToLogIn}"
-                FontSize="Title"
-                FontAttributes="Bold"
-                Margin="0,14,0,21"/>
-            <Label
+    <StackLayout 
+            Padding="7, 0, 7, 20">
+        <ScrollView
+            VerticalOptions="FillAndExpand">
+            <StackLayout
+                VerticalOptions="FillAndExpand">
```

Comment:
Was moving things around and trying different approaches to avoid nesting and forgot about it. In the end needed to add one more level to the tree anyway.